--- ./locales/Makefile.am.org	2008-02-14 00:47:56.000000000 +0100
+++ ./locales/Makefile.am	2008-02-14 22:27:31.221778064 +0100
@@ -7,13 +7,13 @@ install-data-local: install-filezilla-ca
 
 install-filezilla-catalogs: allmo
 	for i in @FILEZILLA_LINGUAS@ ; do \
-	  $(mkinstalldirs) $(DESTDIR)$(localedir)/$$i ; \
-	  $(INSTALL_DATA) $$i.mo $(DESTDIR)$(localedir)/$$i/filezilla.mo ; \
+	  $(mkinstalldirs) $(DESTDIR)$(localedir)/$$i/LC_MESSAGES ; \
+	  $(INSTALL_DATA) $$i.mo $(DESTDIR)$(localedir)/$$i/LC_MESSAGES/filezilla.mo ; \
 	done
 
 uninstall-local:
 	for i in @FILEZILLA_LINGUAS@ ; do \
-	  rm $(DESTDIR)$(localedir)/$$i/filezilla.mo ; \
+	  rm $(DESTDIR)$(localedir)/$$i/LC_MESSAGES/filezilla.mo ; \
 	done
 
 
@@ -22,8 +22,8 @@ uninstall-local:
 
 #install-wxstd-catalogs:
 #	for i in $(WXWIN_LINGUAS) ; do \
-#	  $(mkinstalldirs) $(DESTDIR)$(localedir)/$$i ; \
-#	  $(INSTALL_DATA) wxwin/$$i.mo $(DESTDIR)$(localedir)/$$i/filezilla-wxstd.mo ; \
+#	  $(mkinstalldirs) $(DESTDIR)$(localedir)/$$i/LC_MESSAGES ; \
+#	  $(INSTALL_DATA) wxwin/$$i.mo $(DESTDIR)$(localedir)/$$i/LC_MESSAGES/filezilla-wxstd.mo ; \
 #	done
 
 
--- ./src/interface/FileZilla.cpp.org	2008-02-14 00:47:59.000000000 +0100
+++ ./src/interface/FileZilla.cpp	2008-02-14 22:23:45.254013629 +0100
@@ -578,29 +578,7 @@ bool CFileZillaApp::InitSettingsDir()
 
 bool CFileZillaApp::LoadLocales()
 {
-#ifndef __WXMAC__
-	m_localesDir = GetDataDir(_T("/../locale/*/filezilla.mo"));
-	if (m_localesDir != _T(""))
-	{
-		if (m_localesDir[m_localesDir.Length() - 1] != wxFileName::GetPathSeparator())
-			m_localesDir += wxFileName::GetPathSeparator();
-
-		m_localesDir += _T("../locale");
-	}
-	else
-	{
-		m_localesDir = GetDataDir(_T("/locales/*/filezilla.mo"));
-		if (m_localesDir != _T(""))
-		{
-			if (m_localesDir[m_localesDir.Length() - 1] != wxFileName::GetPathSeparator())
-				m_localesDir += wxFileName::GetPathSeparator();
-
-			m_localesDir += _T("locales");
-		}
-	}
-#else
-	m_localesDir = wxStandardPaths::Get().GetDataDir() + _T("/locales");
-#endif
+	m_localesDir = _T("/usr/share/locale/");
 
 	if (m_localesDir != _T(""))
 	{
--- ./src/interface/optionspage_language.cpp.org	2008-02-14 00:47:57.000000000 +0100
+++ ./src/interface/optionspage_language.cpp	2008-02-14 22:25:47.313341533 +0100
@@ -36,7 +36,7 @@ bool COptionsPageLanguage::LoadPage()
 	wxString locale;
 	for (bool found = dir.GetFirst(&locale); found; found = dir.GetNext(&locale))
 	{
-		if (!wxFileName::FileExists(localesDir + locale + _T("/filezilla.mo")))
+		if (!wxFileName::FileExists(localesDir + locale + _T("/LC_MESSAGES/filezilla.mo")))
 			continue;
 
 		wxString name;
